### PR TITLE
WIP Elewis refactor cmds readme

### PR DESCRIPTION
**Pull request by elewis787**

### CODE REVIEW

**Comment by elewis787** [comment]
@jeff1010322 - I would love to get a review on this. 

Things I want to continue to change ( in seperate pr ) 
- Consts need moved/cleaned up... planning on creating a flags files 
- tui build out - still missing a few items in the new tui pkg 
- clean up of commands - I am thinking about creating a command wrapper ( similar to tui ) for verbs 

Would love feed back on the overall layout. 

I haven't had a chance to test a lot of this but it was largely just moving things around ... only functional change was adding the block height look up before submitting requests ( worth making sure I didn't miss a location for that )

File: cmd/resources/configuration.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		Short: "mazzaroth cli configurations and preferences",
+	}
+
+	cfgRootCmd.AddCommand(
+		verbs.Set(),
+		verbs.Add())
+	return cfgRootCmd
+}
+
+func channelPrompt() (*cfg.ChannelCfg, error) {
```
Comment (jeff1010322) [inline cmd/resources/configuration.go]: Looks like this isn't used anymore.  Its using the one in `add.go`

File: cmd/verbs/initalize.go [hunk 2]
```diff
@@ -12,17 +12,26 @@ import (
...
+	version       = `0.0.1`
+	pubKeyLength  = 32
+	privKeylength = 64
+)
 
-	cfgInitCmd := &cobra.Command{
+func Init() *cobra.Command {
+	init := &cobra.Command{
 		Use:   "init",
+		Short: "initalize resources",
```
Comment (jeff1010322) [inline cmd/verbs/initalize.go]: "initialize" is misspelled in a few areas including in the file name.

File: cmd/verbs/lookup.go [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+		return lookup
+	default:
+		return lookup
+	}
+}
+
+func lookupAbi() *cobra.Command {
+	abi := &cobra.Command{
+		Use:   "abi",
+		Short: "return the application binary interfacem (ABI) for a channel",
```
Comment (jeff1010322) [inline cmd/verbs/lookup.go]: Remove the `m` on `interfacem`

**Review by jeff1010322 — COMMENTED** [review]
I built and tested the commands and everything seems to be working for me.  There were a few verbs that are not being added to the commands currently: `delete.go`, `show.go`, and `list.go` 
For the layout, the only thing that seems odd to me in the cmd layout is the separation between resources and verbs in parallel packages.  Like would it make sense to make the resources each their own package and include the verbs under the specific resource package even if that means there may be an occasional duplicate verb file for different resources?

Something like this:
```
- /cmd
   - /channel
      - channel.go
      - delete.go
      - list.go
      - lookup.go
      - execute.go
      - pause.go
   - /configuration
      - configuration.go
      - add.go
      - initialize.go
      - set.go
      - show.go
```

**Comment by elewis787** [comment]
> I built and tested the commands and everything seems to be working for me.  There were a few verbs that are not being added to the commands currently: `delete.go`, `show.go`, and `list.go` 
> 
> For the layout, the only thing that seems odd to me in the cmd layout is the separation between resources and verbs in parallel packages.  Like would it make sense to make the resources each their own package and include the verbs under the specific resource package even if that means there may be an occasional duplicate verb file for different resources?
> 
> 
> 
> Something like this:
> 
> ```
> 
> - /cmd
> 
>    - /channel
> 
>       - channel.go
> 
>       - delete.go
> 
>       - list.go
> 
>       - lookup.go
> 
>       - execute.go
> 
>       - pause.go
> 
>    - /configuration
> 
>       - configuration.go
> 
>       - add.go
> 
>       - initialize.go
> 
>       - set.go
> 
>       - show.go
> 
> ```

I thought about this originally... A
few things 
- I wanted to be explicit about the resource/verb pkgs to show clearly how new commands should be added and thought about
- I feel this structure provides less code duplication going forward as more resources are added ( I.e the project resource we discussed ) 
- I want to limit the amount of duplication even if it is just at the top level verb command 
- I had the verbs pkg under resources but felt that didn't make sense when the root cmd could have verbs ... I.e show 

That said - I'm not in love with it. 

Let's discuss this morning and I'll chat through what else I'm thinking.

**Review by jeff1010322 — APPROVED** [review]

**Pull request merged**